OutOfSchool/OutOfSchool.DataAccess/Enums/OperationWithObjectEntityType.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace OutOfSchool.Services.Enums;
+
+[JsonConverter(typeof(StringEnumConverter))]
```

Comment:
as this is part of internal job management and not exposed to outside world do we need to have string conversion here?:)

Reply:
I thought in the future we'll make a controller for ours front's tasks )))) But, OK.